universal-login-relayer/lib/core/utils/utils.ts

```diff
@@ -57,5 +57,8 @@ export const getKeyFromHashAndSignature = (messageHash: string, signature: strin
 export const createPendingMessage = (walletAddress: string) : PendingMessage => ({
   walletAddress,
   collectedSignatureKeyPairs: [],
-  transactionHash: null
+  transactionHash: null,
+  error: null,
+  message: null,
```

Comment:
1. message -> put a message here
2. Use the message to calculate finalMessage moment before execution
- extract getMessageWithSignatures to utils: `getMessageWithSignatures(SignedMessage, SignatureKeyPairs [])`
- MessageQueueService use getMessage to get SignedMessage and KeyPairs
3. Remove `addSignedMessage` & `getSignedMessage`
4. getNext() returns message so that we don't need to use getMessage in MessageQueueService